### PR TITLE
Fixes for Linux build

### PR DESCRIPTION
**Pull request by thaytan**
Adds a dep on the system libudev for libusb, but I think that's OK. It's should be available everywhere.

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -82,26 +83,26 @@ if platform == 'linux':
...
-#    sources.append(libusb_path + "libusb/os/poll_windows.c")
-#    sources.append(libusb_path + "libusb/os/threads_windows.c")
-#    env,Append(CPPDEFINES=["OS_WINDOWS"])
-#elif platform == 'osx':
-#    sources.append(libusb_path + "libusb/os/darwin_usb.c")
-#    sources.append(libusb_path + "libusb/os/poll_posix.c")
-#    sources.append(libusb_path + "libusb/os/threads_posix.c")
-#   env.Append(CPPDEFINES=["OS_DARWIN"])
+    if platform == 'linux':
+    #    sources.append(libusb_path + "libusb/os/linux_netlink.c")
```
Comment (BastiaanOlij) [inline SConstruct:87]: This if is now double, it's inline with the previous if platform == 'linux' :)

File: SConstruct [hunk 2]
```diff
@@ -82,26 +83,26 @@ if platform == 'linux':
...
+        sources.append(libusb_path + "libusb/os/linux_usbfs.c")
+        sources.append(libusb_path + "libusb/os/linux_udev.c")
+        sources.append(libusb_path + "libusb/os/events_posix.c")
+        sources.append(libusb_path + "libusb/os/threads_posix.c")
+        env.Append(CPPDEFINES=["OS_LINUX", "USE_UDEV", "HAVE_LIBUDEV"])
+        env.Append(LIBS = ['udev'])
+    #elif platform == 'windows':
```
Comment (BastiaanOlij) [inline SConstruct:94]: and I think we should just remove this commented out code seeing we're not using libusb on windows and osx

Comment (thaytan) [reply SConstruct]: Done

File: SConstruct (hunk 1 above)
Comment (thaytan) [reply SConstruct]: Fixed.

**Pull request merged**